src/etc/inc/system.inc

```diff
@@ -190,7 +190,7 @@ function system_resolvconf_generate($verbose = false)
         $dnsgw = "dns{$dnscounter}gw";
         if (isset($syscfg[$dnsgw])) {
             $gwname = $syscfg[$dnsgw];
-            if (($gwname <> "") && ($gwname <> "none")) {
+            if (($gwname != "") && ($gwname != "none")) {
```

Comment:
taking this as an example, all lines touched should be further tweaked if possible. double quoted strings (") are slightly slower than single quoted ones ('), in all trivial cases a replacement is recommended (trivial = where no string interpolation or escaping like \n takes place)

Reply:
I don't really like touching too many unrelated things within a commit. Especially since the code base is full of these. Also, checking 100 separate places for single/double quotes is not really nice to do.

I'd rather tackle strings in another PR - I'd also be able to address more of them.

Reply:
I don't see a real argument against doing it only for the lines already touched here. At some point I'll have to veto cleanups because the code base is being treated with automated cleanups lacking functional testing. It's far better to clean up code while working on a particular subsystem to ensure integrity. As a hybrid approach, cleanup PRs could still be small and address multiple things in a self-contained way at the same time.

As a general cautionary tale, all well-intentioned clean ups can end in bugs eventually. We just don't know when and how they'll surface.

Reply:
I have done it, but not because I agree with you. This has cost more time than necessary, while only addressing a handful of trivial strings. It's especially jarring in wizard.php, where a find-and-replace throughout would have resulted in a cleaner file, while taking less time.